jobs/kubelet/spec

```diff
@@ -30,3 +30,8 @@ provides:
 - name: kubernetes-workers
   type: kubernetes-workers
 
+# for ha-proxy tcp worker linking
+# the 'ssh_proxy' type must match the 'tcp_backend' link
+# see https://github.com/cloudfoundry-incubator/haproxy-boshrelease/blob/master/jobs/haproxy/spec
+- name: worker_haproxy
+  type: ssh_proxy
```

Comment:
They came in from https://github.com/cloudfoundry-incubator/kubo-release/commit/2c93f615e9556befdb5a2d7356b5962c602c9c48. I'll need to clean it up before the merge.